.github/workflows/ci.yml

```diff
@@ -98,11 +149,13 @@ jobs:
         run: yarn test:script
       - name: Package application
         run: yarn run package
-        if: ${{ matrix.os == 'ubuntu-20.04' && matrix.arch == 'x64' }}
+        if: ${{ matrix.friendlyName == 'Ubuntu' }}
+        env:
+          npm_config_arch: ${{ matrix.arch }}
```

Comment:
maybe we should consider setting `npm_config_arch` in the global env instead of each step.

it isn't required in any of the steps where it currently isn't defined but it does mean not having to worry about it not being there

Reply:
033fb7150054c641340527f87d0057d13dc46439